docs/modules/ROOT/partials/setup_terraform_exoscale.adoc

```diff
@@ -0,0 +1,51 @@
+. Setup Terraform
++
+Prepare terraform
++
+[source,console]
+----
+# Set terraform image and tag to be used
+tf_image=$(\
+  yq eval ".parameters.openshift4_terraform.images.terraform.image" \
+  dependencies/openshift4-terraform/class/defaults.yml)
+tf_tag=$(\
+  yq eval ".parameters.openshift4_terraform.images.terraform.tag" \
+  dependencies/openshift4-terraform/class/defaults.yml)
+
+# Generate the terraform alias
+alias terraform='docker run -it --rm \
+  -u $(id -u) \
+  -e EXOSCALE_API_KEY \
+  -e EXOSCALE_API_SECRET \
+  -e TF_VAR_lb_exoscale_api_key \
+  -e TF_VAR_lb_exoscale_api_secret \
+  -e AWS_ACCESS_KEY_ID \
+  -e AWS_SECRET_ACCESS_KEY \
+  -e AWS_S3_ENDPOINT \
```

Comment:
These are not required when following the rest of the guide to use the GitLab backend